### PR TITLE
Implementing square command

### PR DESCRIPTION
**Pull request by louvreaux**
YO, CHECK DIZZ OUT!!!!1

### CODE REVIEW

File: python/lust.py [hunk 1]
```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
...
+      return
+
+    print(self.__calculate_square(argument))
+
+  def print_help(self):
+    print(" square <integer>")
+    print("   Calculates the square of <integer>.")
+
+  def __calculate_square(self, argument):
+    # Hmmm...
```
Comment (wmlsmm) [inline python/lust.py]: hmmm is not enough

File: python/lust.py [hunk 2]
```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
       result *= i
     return result
 
+
+class SquareCommand(LustObject):
+  def handle(self, arguments):
+    try: argument = int(arguments[0])
+    except (ValueError, IndexError):
+      print("fact: could not read integer argument.")
```
Comment (wmlsmm) [inline python/lust.py]: cmon rly fact

File: python/lust.py [hunk 3]
```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
...
+      return
+
+    print(self.__calculate_square(argument))
+
+  def print_help(self):
+    print(" square <integer>")
+    print("   Calculates the square of <integer>.")
+
+  def __calculate_square(self, argument):
+    # Hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm...
```
Comment (wmlsmm) [inline python/lust.py:47]: ill think about it

**Review by wmlsmm — CHANGES_REQUESTED** [review]

File: python/lust.py (hunk 3 above)
Comment (louvreaux) [reply python/lust.py:47]: pls

**Comment by louvreaux** [comment]
GIT gud

**Pull request merged**